modules/ROOT/pages/alerts-config.adoc

```diff
@@ -5,17 +5,61 @@ endif::[]
 
 Depending on your subscription level, you can configure different types of alerts:
 
-* Operational alerts for APIs through API Manager
-* Operational alerts for the runtime through Runtime Manager
-* Alerts for graphs in custom dashboards in Anypoint Monitoring.
+* Basic alerts for servers and Mule apps
++
+Limit: Up to 50 for users who do not have a Titanium subscription to Anypoint Platform. If you have a Titanium subscription, the limit is 100 basic alerts, but the limit increases for each vCore. For complete details about subscriptions, contact your customer support representative. See also https://www.mulesoft.com/anypoint-pricing[Anypoint Platform pricing].
```

Comment:
Instead of saying "but the limit increases for each vCore", I'd suggest "and the limits raise to higher levels as more vCores are purchased."

Reply:
```suggestion
Limit per organization: Up to 50 for users who do not have a Titanium subscription to Anypoint Platform. If you have a Titanium subscription, the limit is 100 basic alerts, and the limits increase as you purchase more vCores. For complete details about subscriptions, contact your customer support representative. See also https://www.mulesoft.com/anypoint-pricing[Anypoint Platform pricing].
```